lib/chef/provisioning/aws_driver/aws_taggable.rb

```diff
@@ -0,0 +1,18 @@
+module Chef::Provisioning::AWSDriver
+# This module is meant to be included in a resource that is taggable
+# This will add the appropriate attribute that can be converged by the provider
+# TODO it would be nice to not have two seperate modules (taggable/tagger)
```

Comment:
There are magic ways to cause the include in the Provider class to include stuff in the Resource class too, but honestly I don't think we'd want it--separate is fine. The pain we're seeing here is more that they *are* separate things--two separate classes both have to be modified to handle the noun and verb bit of this work. When we do get rid of providers, the separation pain will go away because there will be none.